### PR TITLE
chore: add validate trigger and correct tip message for limit-count plugin

### PR DESCRIPTION
**Pull request by okaybase**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**
add validate trigger and correct tip message for limit-count

- [x] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

fix/resolve

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by liuxiran** [comment]
Nice improve for UE~!

@okaybase would you please add corresponding test case for your feature? 

That will help us merge your pr smoothly, thanks

**Review by iamayushdas — APPROVED** [review]

File: web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js [hunk 1]
```diff
@@ -46,7 +46,9 @@ context('Create and delete consumer with limit-count plugin form', () => {
     consumerName: 'test_consumer',
     description: 'desc_by_autotest',
     createConsumerSuccess: 'Create Consumer Successfully',
-    deleteConsumerSuccess: 'Delete Consumer Successfully'
+    deleteConsumerSuccess: 'Delete Consumer Successfully',
+    redisClusterName: 'Please Enter redis_cluster_name',
```
Comment (iamayushdas) [inline web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js:50]:
```suggestion
    redisClusterName: 'Please Enter name of Redis Cluster',
```

File: web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js [hunk 2]
```diff
@@ -46,7 +46,9 @@ context('Create and delete consumer with limit-count plugin form', () => {
...
+    redisClusterNode: 'Please Enter redis_cluster_node',
```
Comment (iamayushdas) [inline web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js:51]:
```suggestion
    redisClusterNode: 'Please Enter node for Redis Cluster',
```

File: web/cypress/integration/route/create-route-with-limit-count-plugin-form.spec.js [hunk 3]
```diff
@@ -51,6 +51,8 @@ context('Create and delete route with limit-count form', () => {
     submitSuccess: 'Submit Successfully',
     port: '80',
     weight: 1,
+    redisClusterName: 'Please Enter redis_cluster_name',
```
Comment (iamayushdas) [inline web/cypress/integration/route/create-route-with-limit-count-plugin-form.spec.js:54]: ditto

**Review by iamayushdas — CHANGES_REQUESTED** [review]

**Comment by okaybase** [comment]
> Nice improve for UE~!
> 
> @okaybase would you please add corresponding test case for your feature?
> 
> That will help us merge your pr smoothly, thanks

the test cases added. 
thanks for your review~ @liuxiran @juzhiyuan

File: web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js (hunk 1 above)
Comment (okaybase) [reply web/cypress/integration/consumer/create-consumer-with-limit-count-plugin-form.spec.js:50]: i can understand your suggestion, and thanks for your review. 
Only the test contents need consistent with the required tips contents, and the style of all the tips contents for the plugin is like this(redis_cluster_name).

**Comment by juzhiyuan** [comment]
cc @LiteSun @guoqqqi to have a review

**Review by guoqqqi — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**